helm-charts/chatqna/gaudi-values.yaml

```diff
@@ -9,25 +9,20 @@ replicaCount: 1
 
 image:
   repository: opea/chatqna:latest
+  uirepository: opea/chatqna-ui:latest
   pullPolicy: IfNotPresent
   # Overrides the image tag whose default is the chart appVersion.
   # tag: "1.0"
-
+externalIP: 192.168.0.113
+# chatqna port: 30888
+# kubectl port-forward --address 0.0.0.0 svc/chatqna 30888:8888 &
+# dataprep port: 30452
+# UI port: 30173
 port: 8888
 service:
   type: ClusterIP
   port: 8888
 
-securityContext:
-  allowPrivilegeEscalation: false
-  runAsNonRoot: true
-  runAsUser: 1000
-  capabilities:
-    drop:
-    - ALL
-  seccompProfile:
-    type: RuntimeDefault
```

Comment:
why remove default securityContext here?

Reply:
I was working with an old version and these values was overwritten(Include the volume name change).
Will restore them in next revision.